src/redux/actions/creators/profileActions.js

```diff
@@ -54,28 +54,33 @@ export const removeVideo = (video, videoType, uid) => {
   if (videoType === 1) {
     return {
       type: REMOVE_PROFILE_VIDEO,
-      payload: profileRef.update({ youtubeVideo: video })
+      payload: profileRef.update({ video: video })
+    }
+  } else {
+    return {
+      type: REMOVE_PROFILE_VIDEO,
+      payload: profileRef.update({ video: video })
     }
-  }
-  return {
-    type: REMOVE_PROFILE_VIDEO,
-    payload: profileRef.update({ vimeoVideo: video })
   }
 }
 
 export const editVideo = (video, videoType, newTitle, newUrl, uid) => {
   const newVideo = [...video, { title: newTitle, url: newUrl }]
   const profileRef = firebase.database().ref(`/userProfiles/${uid}`)
+  console.log("Video Type: ", videoType)
```

Comment:
Stray console.log

Reply:
Removed these and pushed. Merge it whenever you can